letsencrypt-auto-source/Dockerfile.precise

```diff
@@ -0,0 +1,31 @@
+# For running tests, build a docker image with a passwordless sudo and a trust
+# store we can manipulate.
+
+FROM ubuntu:precise
+
+# Add an unprivileged user:
+RUN useradd --create-home --home-dir /home/lea --shell /bin/bash --groups sudo --uid 1000 lea
+
+# Install pip, sudo, openssl, and nose:
+RUN apt-get update && \
+    apt-get -q -y install python-pip sudo openssl && \
+    apt-get clean
+RUN pip install nose
+
+# Let that user sudo:
+RUN sed -i.bkp -e \
+      's/%sudo\s\+ALL=(ALL\(:ALL\)\?)\s\+ALL/%sudo ALL=NOPASSWD:ALL/g' \
+      /etc/sudoers
+
+RUN mkdir -p /home/lea/letsencrypt/letsencrypt
```

Comment:
Unless I'm missing something, we don't need and never use the inner `letsencrypt` directory. Additionally, this should probably be renamed `certbot` like the Trusty Dockerfile was. (I know you wrote this before the rename.)